editor.planx.uk/src/@planx/components/Confirmation/Public.tsx

```diff
@@ -19,11 +20,38 @@ export type Props = PublicProps<Confirmation>;
 export default function ConfirmationComponent(props: Props) {
   const [data, setData] = useState<QuestionAndResponses[]>([]);
 
-  const [sessionId, saveToEmail, $public] = useStore((state) => [
-    state.sessionId,
-    state.saveToEmail,
-    state.$public,
-  ]);
+  const [sessionId, saveToEmail, $public, passport, govUkPayment, flowName] =
```

Comment:
Quite a few Confirmation related changes in this PR - expecting these to go away once `dp/vite` has rebased to `main` 👍